mw-webapp/src/component/editableText/EditableText.module.scss

```diff
@@ -6,8 +6,8 @@ $editableTextMinWidth: 40px;
 .editableText {
   overflow: hidden;
   width: $editableTextWidth;
-
-  input {
-    min-width: $editableTextMinWidth;
+  input[type="number"]::-webkit-inner-spin-button {
+    margin: 0;
+    -webkit-appearance: none;
   }
```

Comment:
1. Could you please check is it work in Firefox?
2. Is it possible to use className and add -webkit-inner-spin-button  instead of adding property to all number inputs?
3. Can you please explain difference between  -webkit-inner-spin-button  and -webkit-outer-spin-button

Reply:
1. Done

Reply:
2. Done

Reply:
3. No